tests/test_filter_azimuthal/test_filter_azimuthal.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+
+import os
+import sys
+sys.path.insert(0, os.pardir)
+from testing_harness import TestHarness, PyAPITestHarness
+import openmc
+
+class FilterAzimuthalTestHarness(PyAPITestHarness):
+    def _build_inputs(self):
+        filt1 = openmc.Filter(type='azimuthal',
+                              bins=(-3.1416, -1.8850, -0.6283, 0.6283, 1.8850,
+                                    3.1416))
+        tally1 = openmc.Tally(tally_id=1)
+        tally1.add_filter(filt1)
+        tally1.add_score('flux')
+        tally1.estimator = 'tracklength'
+
+        tally2 = openmc.Tally(tally_id=2)
+        tally2.add_filter(filt1)
+        tally2.add_score('flux')
+        tally2.estimator = 'analog'
+
+        filt3 = openmc.Filter(type='azimuthal', bins=(5))
```

Comment:
`bins=(5)` doesn't actually pass a tuple. You'd have to do `bins=(5,)`, but in any event `bins=5` is acceptable.